src/gsudo.Installer/Product.wxs

```diff
@@ -30,9 +30,9 @@
 
     <InstallExecuteSequence>
       <!-- https://stackoverflow.com/a/17608049/97471 -->
+      <Custom Action="CreateSudoCurrentFolderLink" Before="InstallFinalize" Condition="NOT Installed" />

```

Comment:
Have to create the `Current` junction directory before we create the `sudo.exe` symlink.